sql-db/panache-flyway/src/test/java/io/quarkus/qe/sqldb/panacheflyway/DataSourceIT.java

```diff
@@ -27,6 +24,6 @@ public void shouldDataSourceBeProperlyConfigured() {
     }
 
     private static final RequestSpecification dataSourcePath() {
-        return given().accept(MediaType.APPLICATION_JSON).contentType(ContentType.JSON).when().basePath(DATA_SOURCE_PATH);
+        return given().when().basePath(DATA_SOURCE_PATH);
```

Comment:
This was wrong and using reactive fails which I think it's expected. It's now corrected.